RecoMTD/TrackExtender/plugins/TrackExtenderWithMTD.cc

```diff
@@ -379,6 +504,9 @@ class TrackExtenderWithMTDT : public edm::stream::EDProducer<> {
   edm::EDPutToken pathLengthOrigTrkToken;
   edm::EDPutToken tmtdOrigTrkToken;
   edm::EDPutToken sigmatmtdOrigTrkToken;
+  edm::EDPutToken tofpiOrigTrkToken;
+  edm::EDPutToken tofkOrigTrkToken;
+  edm::EDPutToken tofpOrigTrkToken;
```

Comment:
it could be a good opportunity to update the data members to have consistent naming pattern (with a preferred trailing `_` in the names)
```suggestion
  edm::EDPutToken tofpOrigTrkToken_;
```

Reply:
done